scripts/trifingerpro_post_submission.py

```diff
@@ -533,10 +621,22 @@ def main():
     # terminate the robot
     del robot
 
+    print("Check cameras")
+    camera_observations = record_camera_observations(
```

Comment:
How can I be sure that the object is at rest at this point? Is there some time interval at the end of the replayed trajectory where the fingers don't move?

Reply:
Deleting the `robot` object in the line above triggers the shutdown of the robot, i.e. the fingers move to the rest position.  So when the camera is started, the robot should already have stopped moving and hasn't been in contact with the object anymore for a few seconds.

Reply:
Makes sense. Thank you for the explanation!